datasets/cloud_storage_geo_index/pipelines/cloud_storage_geo_index/pipeline.yaml

```diff
@@ -0,0 +1,293 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+  - type: bigquery_table
+    table_id: landsat_index
+    description: "Landsat_Index Dataset"
+  - type: bigquery_table
+    table_id: sentinel_2_index
+    description: "Sentinel_2_Index Dataset"
+
+dag:
+  airflow_version: 2
+  initialize:
+    dag_id: cloud_storage_geo_index
+    default_args:
+      owner: "Google"
+      depends_on_past: False
+      start_date: '2021-03-01'
+    max_active_runs: 1
+    schedule_interval: "0 1 0 0 6"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "GKECreateClusterOperator"
```

Comment:
Any chance we can use individual kube pods?

Reply:
Not sure what you are asking.  Could you please elaborate?

Reply:
We're creating a cluster here instead of using the `KubernetesPodOperator`. What would be the context for using an ephemaral cluster (create then delete after used) over kube pods?

Reply:
KubernetesPodOperator would fail on resources.  I understand that we have limited number of clusters but each of these clusters handles large files and that is resource intensive.  Just testing locally brought my machine to a standstill.